### PR TITLE
Address `DiscoverySelectors` conflict

### PR DESCRIPTION
**Pull request by PetrMc**
…suecomment-2083122552

*Issue #, if available:*

*Description of changes:*

Added "DiscoverySelectors"

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by elamaran11** [comment]
@PetrMc Thankyou for such a quick turnaround. We did a revert so your changes from last week is missing in the PR. Please sync your fork and then push the changes.

**Comment by PetrMc** [comment]
sorry missed it. Should be good to go now.

File: eks-anywhere-common/Addons/Partner/Solo.io/solo-istiod.yaml [hunk 1]
```diff
@@ -14,4 +13,13 @@ spec:
         name: solo-istiod-charts
         namespace: flux-system
       version: 1.18.3-eks-a
+  values:
+    revision: 1-18-3-solo
+    global:
+      istioNamespace: solo-istio-system       
+    meshConfig:
```
[the remark this answers is not in the document]
Comment (elamaran11) [reply eks-anywhere-common/Addons/Partner/Solo.io/solo-istiod.yaml:20]: @PetrMc Would you be able to make this change for both service mesh to work together in same cluster.

**Review by elamaran11 — CHANGES_REQUESTED** [review]
@PetrMc Could you make this change?

**Comment by PetrMc** [comment]
@elamaran11 @zirain - the requested change is applied, please let me know if anything else is needed.

**Comment by elamaran11** [comment]
> @elamaran11 @zirain - the requested change is applied, please let me know if anything else is needed.

Thankyou @PetrMc

**Review by mikemcd3912 — APPROVED** [review]
Thank you @PetrMc for the quick updates on this! Everything is working as expected and both PR's are co-existing on the cluster with that new edit applied. LGTM 

EKSA: 

- VMware (K8s 1.28, Bottlerocket 1.15): Pods deploy and reach a ready state, tester completes successfully on first attempt
- Outposts (K8s 1.27, Amazon Linux 2): Pods deploy and reach a ready state, tester completes successfully on first attempt
- Baremetal (K8s 1.27, Ubuntu 20.04.6 LTS): Pods deploy and reach a ready state, tester completes successfully on first attempt

EKS:

- EKS (K8s 1.28, ARM Bottlerocket 1.19): Pods deploy and reach a ready state, tester completes successfully on first attempt
- EKS (K8s 1.28, ARM Amazon Linux 2): Pods deploy and reach a ready state, tester completes successfully on first attempt
- EKS (K8s 1.28, x86 Amazon Linux 2): Pods deploy and reach a ready state, tester completes successfully on first attempt
- EKS (K8s 1.28, x86 Bottlerocket): Pods deploy and reach a ready state, tester completes successfully on first attempt

**Review by elamaran11 — APPROVED** [review]
LGTM

**Comment by elamaran11** [comment]
Looks like your deployment `istiod-1-18-3-solo` is failing.
Specifically, it looks like these pods are failing: 
* Pod: `istiod-1-18-3-solo-5f996795d8-czvmd`.
----

**Comment by PetrMc** [comment]
@elamaran11 / @mikemcd3912 is there any action that are required on my end?

**Pull request merged**